pkg/domain/infra/abi/play.go

```diff
@@ -964,3 +978,53 @@ func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, _ e
 
 	return reports, nil
 }
+
+// playKubeSecret allows users to create and store a kubernetes secret as a podman secret
+func (ic *ContainerEngine) playKubeSecret(secret *v1.Secret) (*entities.SecretCreateReport, error) {
+	r := &entities.SecretCreateReport{}
+
+	// Create the secret manager before hand
+	secretsManager, err := ic.Libpod.SecretsManager()
+	if err != nil {
+		return nil, err
+	}
+
+	// only store the data!!! if we store the entire secret marshaled,
+	// we cannot unmarshal it later. (yaml.unmarshal does not like string --> byte conversion)
```

Comment:
Does this comment still hold true? it looks like you're marshaling the entire *v1.Secret, not just the data